test/NVQPP/conditional_sample.cpp

```diff
@@ -7,9 +7,11 @@
  ******************************************************************************/
 
 // clang-format off
-// RUN: nvq++ --enable-mlir %s -o %t && %t
-// RUN: nvq++ --target quantinuum --emulate %s -o %t && %t
 // RUN: nvq++ -std=c++17 --enable-mlir %s -o %t
+// RUN: if [ $(echo %cpp_std | cut -c4- ) -ge 20 ]; then \
+// RUN:   nvq++ --enable-mlir %s -o %t && %t; \
+// RUN: fi
```

Comment:
Why do we need this special `if` check here? I think this could be simplified. (I checked my suggestion locally with `-std=cpp17` and `-std=cpp20`.)
```suggestion
// RUN: nvq++ %cpp_std --enable-mlir %s -o %t && %t
```
Or if you want to keep the explicit `-std=c++17` in there:
```suggestion
// RUN: nvq++ -std=c++17 --enable-mlir %s -o %t
// RUN: nvq++ %cpp_std --enable-mlir %s -o %t && %t
```

Reply:
My thinking was exactly to keep the explicit c++17 so that we also have some minimal c++17 tests when running in our dev images. And the if-statement was only there to not run the same exact test twice.

Reply:
> My thinking was exactly to keep the explicit c++17 so that we also have some minimal c++17 tests when running in our dev images. And the if-statement was only there to not run the same exact test twice.

OK. That's what I was suspecting, but I wasn't sure.